test/jasmine/assets/mouse_event.js

```diff
@@ -1,11 +1,16 @@
-module.exports = function(type, x, y) {
-    var options = {
+module.exports = function(type, x, y, opts) {
+    var fullOpts = {
         bubbles: true,
         clientX: x,
         clientY: y
     };
 
+    // https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent
+    if(opts) {
```

Comment:
I wrote it that way so that it is easily expendable to other `opts`.

But, yeah, let's cross that bridge only when we get there.